eva-lib/src/main/java/uk/ac/ebi/eva/lib/entities/Taxonomy.java

```diff
@@ -46,13 +46,21 @@
 })
 @NamedNativeQueries({
         @NamedNativeQuery(
-                name = "Taxonomy.getSpecies",
+                name = "Taxonomy.getBrowsableSpecies",
                 query = "select distinct(assembly.*), taxonomy.* " +
                         "from assembly left join browsable_file bf on assembly.assembly_set_id=bf.assembly_set_id " +
+                        " and assembly.assembly_accession = bf.loaded_assembly " +
                         "left join dbsnp_assemblies dbs on assembly.assembly_set_id = dbs.assembly_set_id " +
                         "join taxonomy on assembly.taxonomy_id=taxonomy.taxonomy_id " +
                         "where (bf.loaded = true and bf.deleted = false) or (dbs.loaded = true)",
                 resultSetMapping = "assembly"
+        ),
+        @NamedNativeQuery(
+                name = "Taxonomy.getAccessionedSpecies",
+                query = "select distinct(assembly.*), taxonomy.* " +
+                        "from assembly join taxonomy on assembly.taxonomy_id=taxonomy.taxonomy_id " +
+                        "where assembly_in_accessioning_store = true",
+                resultSetMapping = "assembly"
```

Comment:
Uses the new "assembly" view definition introduced as part of the [schema changes](https://www.ebi.ac.uk/panda/jira/browse/EVA-1376?focusedCommentId=273658&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-273658)